pandas/core/groupby/ops.py

```diff
@@ -969,7 +985,7 @@ def _cython_operation(
         """
         assert kind in ["transform", "aggregate"]
 
-        cy_op = WrappedCythonOp(kind=kind, how=how)
+        cy_op = WrappedCythonOp(grouper=self, kind=kind, how=how)
```

Comment:
@jbrockmendel - It would suffice to pass `self.has_dropped_na` instead of the full grouper here, wasn't sure if there was a preference one way or the others.

Reply:
I'd much rather just has_dropped_na be part of WrappedCythonOop's state